staging/src/k8s.io/kube-scheduler/config/v1/doc.go

```diff
@@ -14,8 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package v1 is the v1 version of the kube-scheduler config API
 // +k8s:deepcopy-gen=package
 // +k8s:openapi-gen=true
 // +groupName=kubescheduler.config.k8s.io
-
```

Comment:
Other api packages have a blank line here intentionally. Is dropping it here correct?

Reply:
xref https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/apps/v1/doc.go

Reply:
It was intentional. If this is not what we want, I can submit a PR to ahmetb/gen-crd-api-reference-docs to use all comments instead of just the package doc comment.